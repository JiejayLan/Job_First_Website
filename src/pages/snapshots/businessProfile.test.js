import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import BusinessProfile from '../businessProfile';

describe('Business profile screenshot test', () => {
    test('Business profile should render correctly', () => {
        const renderer = new ReactShallowRenderer();
        renderer.render(<BusinessProfile />);
        expect(renderer.getRenderOutput()).toMatchSnapshot();
    });
});