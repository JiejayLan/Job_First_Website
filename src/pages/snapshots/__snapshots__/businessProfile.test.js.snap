// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Business profile screenshot test Business profile should render correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
      "margin": "auto",
      "width": "60%",
    }
  }
>
  <h1>
    Company: aplipap;
  </h1>
  <profileCard />
  <h2>
    Location:NY USA
  </h2>
  <h1>
    About us
  </h1>
  <div>
    At AWS, we believe nothing should stand in a builder’s way, and dreams never have to turn off. We’re a company of builders, innovators and creators. Our employees experience unparalleled ownership and impact on our products, and are empowered to innovate and deliver. If you want to build the future with AWS, we’d love to hear from you.
  </div>
  <h2>
    TimeLine
  </h2>
  <timeLine />
</div>
`;
